README.rst

```diff
@@ -1,29 +1,41 @@
-aiofreepybox
-============
+freebox-api
+===========
 
 |PyPI| |Python Version| |License|
 
 |Read the Docs| |Tests| |Codecov|
 
 |pre-commit| |Black|
 
-.. |PyPI| image:: https://img.shields.io/pypi/v/aiofreepybox.svg
-   :target: https://pypi.org/project/aiofreepybox/
+.. |PyPI| image:: https://img.shields.io/pypi/v/freebox-api
+.svg
+   :target: https://pypi.org/project/freebox-api
+/
    :alt: PyPI
-.. |Python Version| image:: https://img.shields.io/pypi/pyversions/aiofreepybox
-   :target: https://pypi.org/project/aiofreepybox
+.. |Python Version| image:: https://img.shields.io/pypi/pyversions/freebox-api
+
+   :target: https://pypi.org/project/freebox-api
+
    :alt: Python Version
-.. |License| image:: https://img.shields.io/pypi/l/aiofreepybox
+.. |License| image:: https://img.shields.io/pypi/l/freebox-api
+
    :target: https://opensource.org/licenses/GPL-3.0
    :alt: License
-.. |Read the Docs| image:: https://img.shields.io/readthedocs/aiofreepybox/latest.svg?label=Read%20the%20Docs
-   :target: https://aiofreepybox.readthedocs.io/
-   :alt: Read the documentation at https://aiofreepybox.readthedocs.io/
-.. |Tests| image:: https://github.com/hacf-fr/aiofreepybox/workflows/Tests/badge.svg
-   :target: https://github.com/hacf-fr/aiofreepybox/actions?workflow=Tests
+.. |Read the Docs| image:: https://img.shields.io/readthedocs/freebox-api
+/latest.svg?label=Read%20the%20Docs
+   :target: https://freebox-api
+.readthedocs.io/
+   :alt: Read the documentation at https://freebox-api
+.readthedocs.io/
+.. |Tests| image:: https://github.com/hacf-fr/freebox-api
+/workflows/Tests/badge.svg
+   :target: https://github.com/hacf-fr/freebox-api
+/actions?workflow=Tests
    :alt: Tests
-.. |Codecov| image:: https://codecov.io/gh/hacf-fr/aiofreepybox/branch/master/graph/badge.svg
-   :target: https://codecov.io/gh/hacf-fr/aiofreepybox
+.. |Codecov| image:: https://codecov.io/gh/hacf-fr/freebox-api
+/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/hacf-fr/freebox-api
+
```

Comment:
I think you replaced `aiofreepybox` by `freebox-api\n`.

`\n` shouldn't be here

Reply:
corrected